ansible/playbook.yml

```diff
@@ -1,29 +1,35 @@
 ---
 - name: Set up training VM
   hosts: local
-  gather_subset: min
+
   roles:
+    - role: initial_setup
+      tags: initial_setup
+      when: initial_setup
+
     - role: catrust
       tags: catrust
       when: catrust
 
-    - role: common        #  <- mandatory
+    - role: common #  <- mandatory
       tags: common
 
+    - role: docker
+      tags: docker
+      when: docker
+
     - role: epics-modules
       tags: epics_modules
-      when: epics_modules
+      when: epics_modules_build
```

Comment:
Why that?

Reply:
because epics-modules is also the list of config items declaring dependencies etc. 

Thus it would always build epics modules. If I set the Boolean to false in all.yml it is overriden as the list.

Happy to consider a different unique name!

Reply:
> because epics-modules is also the list of config items declaring dependencies etc.
> Thus it would always build epics modules. If I set the Boolean to false in all.yml it is overriden as the list.

Duh. I didn't realize that. Don't think I ever built without epics-modules, that's why.
Clearly not enough tests.

> Happy to consider a different unique name!

For me, this is more about the uniformity between the role control variables.
I'd probably rather rename the list than have this one stick out without apparent reason in the playbook.

Reply:
Yes I prefer that too